app/assets/javascripts/client/home/testimonal.js

```diff
@@ -0,0 +1,25 @@
+(function() {
+  'use strict';
+
+  angular.module('PDRClient')
+      .directive('testimonial', testimonial);
+
+  function testimonial() {
+    return {
+      restrict: 'E',
+      transclude: true,
```

Comment:
Muscle memory, for the most part.  It doesn't look like I need it so I'll remove it.